### PR TITLE
Fixed continuous config appending

### PR DESCRIPTION
**Pull request by duanehutchins**
Fixed the issue where the stunnel-pgbouncer.conf, user.txt and pgbouncer.ini files are being repeatedly appended on server restarts, instead of erasing and restarting.
Also fixed the DATABASE_URL parse to not fail on missing passwords.

### CODE REVIEW

**Comment by edmorley** [comment]
On Heroku the filesystem is ephemeral, so previous run's configs won't exist any more.
Was the issue you were seeing when using this buildpack ion other environments?

**Comment by edmorley** [comment]
Ah there's a dupe of this, #70 which gives an example scenario.

**Comment by duanehutchins** [comment]
@gregburek I split this into two separate pull requests.

File: bin/gen-pgbouncer-conf.sh [hunk 1]
```diff
@@ -86,7 +86,7 @@ connect = $DB_HOST:$DB_PORT
 retry = ${PGBOUNCER_CONNECTION_RETRY:-"no"}
 EOFEOF
 
-  cat >> /app/vendor/pgbouncer/users.txt << EOFEOF
+  cat > /app/vendor/pgbouncer/users.txt << EOFEOF
```
Comment (gregburek) [inline bin/gen-pgbouncer-conf.sh]: yeah this is inside the for loop, so it must be append only. Fixed in #89

**Pull request merged**